lib/std/cmath.nim

```diff
@@ -0,0 +1,6 @@
+##[
+Low level wrappers around C math functions.
+]##
+
+proc isnan*(x: float): bool {.importc: "isnan", header: "<math.h>".}
```

Comment:
Put this declaration in ``math.nim`` and don't export it. And name it ``cisnan`` or better.

Reply:
done (with `c_isnan`) to unblock this PR but my plan is https://github.com/nim-lang/RFCs/issues/92#issuecomment-735328291, can we continue this conversation there ? (already in an RFC)